### PR TITLE
chore(Python): add uniform cost search

### PR DESCRIPTION
**Pull request by GAbdulWahab**
**PR Checklist:**
- [x] My submission is formatted according to the guidelines in the [contributing guide](https://github.com/MakeContributions/DSA#contribution-guidelines)
- [x] My addition is on refer on the language README.md file
- [x] My addition does not have a spelling problem
- [x] My submission has a proper and user-friendly description of the algorithm
- [x] My submission has the time complexity of the algorithm
- [ ] My submission has sample input-output of the program (NOT FOR PYTHON)

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

**Other information:**

### CODE REVIEW

**Comment by GAbdulWahab** [comment]
@sridhar-5 
Why my code is attempting ,?

**Review by sridhar-5 — APPROVED** [review]

**Comment by sridhar-5** [comment]
looks good to me.  @ming-tsai we can take this forward. sorry for the late review. was busy.

File: algorithms/Python/searching/Uniform_Cost_Search .py
Comment (sridhar-5) [inline algorithms/Python/searching/Uniform_Cost_Search .py:18]: Please change the  (alphabeticaly) ->to  (alphabetically ) here. It failed because of the spell mistake in the comment. your code is good. please change the spelling.

**Review by sridhar-5 — CHANGES_REQUESTED** [review]

File: algorithms/Python/searching/Uniform_Cost_Search .py
Comment (GAbdulWahab) [reply algorithms/Python/searching/Uniform_Cost_Search .py]: I make it, thank sir

**Review by GAbdulWahab — COMMENTED** [review]
The spelling has been modified

**Review by Arsenic-ATG — CHANGES_REQUESTED** [review]
please also update the corresponding README.md file with link to your program under appropriate heading.

**Comment by GAbdulWahab** [comment]
@Arsenic-ATG sorry I can't understand what should I do can you clarify?

**Comment by GAbdulWahab** [comment]
@Arsenic-ATG I make it but you should accept my code first

**Review by Arsenic-ATG — DISMISSED** [review]

**Comment by GAbdulWahab** [comment]
@Arsenic-ATG  why is it not merged?

**Comment by Arsenic-ATG** [comment]
> @Arsenic-ATG why is it not merged?

After the approval by the a reviewer, it's generally @ming-tsai who finally merge the changes into the main repo ( adds another layer of reviewing over the existing methods ).

----
Looks like you updated the PR by pushing changes [44c9850](https://github.com/MakeContributions/DSA/pull/723/commits/44c9850f9ebda8321bfb752c043a5afb495793ca) and  [5b7325a](https://github.com/MakeContributions/DSA/pull/723/commits/5b7325a3db8ae3c151b8d3a47a2a22bbf89cc6de) in the base branch before the earlier changes before were merged. 
It is generally a good idea to avoid touching the base branch of the PR after a positive approval by a contributor ( and wait for it to get merged first ). You can always create another branch in case you want to work on something else during that period.

File: algorithms/Python/searching/Breadth-first search.py
Comment (ming-tsai) [inline algorithms/Python/searching/Breadth-first search.py:1]: Hi @GAbdulWahab,
Remove this file change from this branch

**Review by Arsenic-ATG — DISMISSED** [review]

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**
